lua/nvim-tree.lua

```diff
@@ -232,6 +232,17 @@ local function setup_autocommands(opts)
       end,
     })
   end
+
+  if opts.actions.set_last_win_as_target then
+    create_nvim_tree_autocmd("BufLeave", {
+      callback = function()
+        local buf_name = vim.api.nvim_buf_get_name(0)
```

Comment:
bufnr / bufname should come from the `ev` argument passed. See `:help nvim_create_autocmd()`